src/app/shared/components/input-field/input-field.component.scss

```diff
@@ -4,4 +4,12 @@
 
 mat-icon:hover{
     cursor: pointer;
-}
\ No newline at end of file
+}
+
```

Comment:
these override the styling changes that were made in the new implementation for mat-form-fields, which was adding a box-shadow in the back of every input text box